bigtable/google/cloud/bigtable/table.py

```diff
@@ -309,6 +309,10 @@ def yield_rows(self, start_key=None, end_key=None, limit=None,
                         specified row(s). If unset, reads every column in
                         each row.
 
+        :type row_set: :class:`row_set.RowSet`
+        :param filter_: (Optional) The row set containing multiple row keys and
+                        row_ranges.
+
```

Comment:
The `row_set` parameter is not used in this method.

Reply:
Forgot to add row_set in parameter list, added now.